apps/omg/test/omg/state/core_test.exs

```diff
@@ -680,23 +680,29 @@ defmodule OMG.State.CoreTest do
   @tag fixtures: [:alice, :state_alice_deposit, :state_empty]
   test "given exit infos in various forms translates to utxo positions",
        %{alice: alice, state_alice_deposit: state, state_empty: state_empty} do
+    # TODO: Fix this !@#%
     # this test checks whether all ways of calling `get_exiting_utxo_positions/2` translates
     # to given exiting utxo positions
-    utxo_pos_exits = [Utxo.position(@blknum1, 0, 0), Utxo.position(@blknum1, 0, 1)]
+    expected_utxo_pos_exits = [{:utxo_position, @blknum1, 0, 0}, {:utxo_position, @blknum1, 0, 1}]
 
-    assert utxo_pos_exits ==
+    utxo_pos_exits = [
+      %{blknum: @blknum1, txindex: 0, oindex: 0},
+      %{blknum: @blknum1, txindex: 0, oindex: 1}
+    ]
```

Comment:
Will come back to re-visit this in a follow up PR, but I think we should move the data structure into a map or struct. I believe this will provide more clarity and mileage then sticking with the various atom-led tuple formats e.g:
*  `{:utxo_position, x, y, z}`
* `{:input_pointer, @input_pointer_type_marker, {x, y, z}`
* `{x, y, z}`

where `x,y,z` respectively is `blknum, txindex, oindex`